mlir/include/mlir/Dialect/OpenMP/Transforms/Passes.h

```diff
@@ -0,0 +1,26 @@
+//===- Passes.h - LLVM Pass Construction and Registration -----------------===//
```

Comment:
```suggestion
//===- Passes.h - MLIR Pass Construction and Registration -------*- C++ -*-===//
```
[nit] `-*- C++ -*-` used used by vim to identifiy what syntax highlighting to use. I don't use it, but it is the way headers comments are formatted.